Shared/Samples/Style symbols from mobile style file/StyleSymbolsFromMobileStyleFileView.SymbolOptionsListView.swift

```diff
@@ -0,0 +1,121 @@
+        
+        /// The view model for the sample.
+        @ObservedObject var model: Model
+        
+        /// The list of all the symbols and their associated data from the symbol style.
+        @State private var symbols: [SymbolDetails] = []
+        
+        var body: some View {
+            VStack {
+                Image(uiImage: model.currentSymbol?.image ?? UIImage())
+                
+                List {
+                    // Create a picker for each symbol category.
+                    ForEach(SymbolCategory.allCases, id: \.self) { category in
+                        Section(category.label) {
+                            Picker("Symbol Names", selection: $model.symbolOptionSelections.categoryKeys[category]) {
+                                ForEach(symbols.filter { $0.category == category.rawValue }, id: \.key) { symbol in
+                                    Label {
+                                        Text(symbol.label)
+                                    } icon: {
+                                        Image(uiImage: symbol.image)
+                                    }
+                                    .tag(symbol.key as String?)
+                                }
+                            }
+                            .pickerStyle(.inline)
+                            .labelsHidden()
+                        }
+                    }
+                    
+                    Section {
+                        ColorPicker("Color", selection: $model.symbolOptionSelections.color)
+                        VStack {
+                            HStack {
+                                Text("Size")
+                                Spacer()
+                                Text(model.symbolOptionSelections.size.formatted(.number.rounded()))
+                                    .foregroundColor(.secondary)
+                            }
+                            Slider(value: $model.symbolOptionSelections.size, in: 20...60, step: 1)
+                        }
+                    }
+                }
+            }
+            .task(id: model.symbolOptionSelections) {
+                // Update the current symbol when an option selection changes.
+                await model.updateCurrentSymbol(displayScale: displayScale)
+            }
+            .task(id: displayScale) {
+                // Update the symbols when the display scale changes.
+                symbols = await getSymbols(displayScale: displayScale)
+                await model.updateCurrentSymbol(displayScale: displayScale)
+            }
+        }
+        
+        /// Gets the symbols from the symbol style.
+        /// - Parameter displayScale: The display scale of the environment for creating symbol swatches.
+        /// - Returns:  An `Array` of `SymbolDetails` which contain a symbol and its associated information.
+        private func getSymbols(displayScale: Double) async -> [SymbolDetails] {
```

Comment:
You could think about renaming this
```suggestion
        private func symbols(forDisplayScale: Double) async -> [SymbolDetails] {
```